### ISSUE TITLE
Defect: Failure in reading a file for the checksum

### ISSUE THREAD

**Issue by uihsnv** [sourceryinstitute/OpenCoarrays#665]
- [x] I am reporting a bug others will be able to reproduce and not asking a question or requesting a new feature.

## System information including:
  - OpenCoarrays Version: **2.7.0**
  - Fortran Compiler: **gfortran 8.3.0**
  - C compiler used for building lib: **gcc 8.3.0**
  - Installation method: **AUR** [build script](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=opencoarrays)
  - All flags & options passed to the installer:
```
    -DCMAKE_INSTALL_PREFIX=/usr \
    -DCMAKE_INSTALL_LIBDIR=lib \
    -DMPI_C_COMPILER=/opt/mpich/bin/mpicc \
    -DMPI_Fortran_COMPILER=/opt/mpich/bin/mpifort
```
  - Output of `uname -a`: **GNU/Linux 5.0.13 x86_64**
  - MPI library being used: **3.3b**
  - Machine architecture and number of physical cores: **4 cores (8 threads)**
  - Version of CMake: **3.14.3**

### What you were trying to do (and why)

Trying to install the software.

### What happened (include command output, screenshots, logs, etc.)

After the build was complete, the error occurred during the packaging:

```
Install the project...
-- Install configuration: "Release"
-- Installing: /tmp/makepkg/opencoarrays/pkg/opencoarrays/usr/share/man/man1/caf.1
-- Installing: /tmp/makepkg/opencoarrays/pkg/opencoarrays/usr/share/man/man1/cafrun.1
-- Installing: /tmp/makepkg/opencoarrays/pkg/opencoarrays/usr/include
-- Installing: /tmp/makepkg/opencoarrays/pkg/opencoarrays/usr/include/opencoarrays.mod
-- Installing: /tmp/makepkg/opencoarrays/pkg/opencoarrays/usr/include/OpenCoarrays-2.7.0_GNU-8.3.0
-- Installing: /tmp/makepkg/opencoarrays/pkg/opencoarrays/usr/include/OpenCoarrays-2.7.0_GNU-8.3.0/opencoarrays.mod
-- Installing: /tmp/makepkg/opencoarrays/pkg/opencoarrays/usr/lib/libopencoarrays_mod.a
-- Installing: /tmp/makepkg/opencoarrays/pkg/opencoarrays/usr/lib/libcaf_mpi.so.3
-- Installing: /tmp/makepkg/opencoarrays/pkg/opencoarrays/usr/lib/libcaf_mpi.so
-- Installing: /tmp/makepkg/opencoarrays/pkg/opencoarrays/usr/lib/libcaf_mpi.a
-- Installing: /tmp/makepkg/opencoarrays/pkg/opencoarrays/usr/bin/cafrun
-- Installing: /tmp/makepkg/opencoarrays/pkg/opencoarrays/usr/bin/caf
-- Installing: /tmp/makepkg/opencoarrays/pkg/opencoarrays/usr/include/libcaf.h
-- Installing: /tmp/makepkg/opencoarrays/pkg/opencoarrays/usr/include/libcaf-gfortran-descriptor.h
-- Installing: /tmp/makepkg/opencoarrays/pkg/opencoarrays/usr/include/libcaf-version-def.h
-- Installing: /tmp/makepkg/opencoarrays/pkg/opencoarrays/usr/lib/cmake/opencoarrays/OpenCoarraysTargets.cmake
-- Installing: /tmp/makepkg/opencoarrays/pkg/opencoarrays/usr/lib/cmake/opencoarrays/OpenCoarraysTargets-release.cmake
-- Installing: /tmp/makepkg/opencoarrays/pkg/opencoarrays/usr/lib/cmake/opencoarrays/OpenCoarraysConfig.cmake
-- Installing: /tmp/makepkg/opencoarrays/pkg/opencoarrays/usr/lib/cmake/opencoarrays/OpenCoarraysConfigVersion.cmake
-- Generating SHA256 checksumed receipt for all installed assets
CMake Error at /tmp/makepkg/opencoarrays/src/OpenCoarrays-2.7.0/cmake/InstallFileManifest.cmake:21 (file):
  file SHA256 failed to read file "/usr/share/man/man1/caf.1": No such file
  or directory
Call Stack (most recent call first):
  cmake_install.cmake:95 (include)


make: *** [Makefile:84: install] Error 1
```

### Step-by-step reproduction instructions to reproduce the error/bug

I'm using the build script linked above.

**Comment by uihsnv** [comment]
The path should be prefixed by the `DESTDIR` environment variable. This is necessary for any distributions that package it. I am not sure if it is the same variable in all distros. This is so in Archlinux.

**Comment by Meriipu** [comment]
Reproducible for me too with this (third-party) portage ebuild
```
# Copyright 1999-2017 Gentoo Foundation
# Distributed under the terms of the GNU General Public License v2

EAPI=6

inherit cmake-utils fortran-2 git-r3
DESCRIPTION="Implementation of coarrays for gfortran"
HOMEPAGE="http://www.opencoarrays.org/"

#SRC_URI="https://github.com/sourceryinstitute/${PN}/releases/download/${PV}/${P}.tar.gz"
SRC_URI=""
EGIT_REPO_URI="https://github.com/sourceryinstitute/${PN}"
EGIT_BRANCH="master"

RESTRICT=mirror

LICENSE="BSD"
SLOT="0"
KEYWORDS="~amd64 ~x86"

DEPEND="virtual/fortran virtual/mpi"
RDEPEND="${DEPEND}"
```
I tried patching the makefile commenting out the manpage generation to see if that was the only issue but doing that it breaks at
```
  file SHA256 failed to read file
  "/usr/include/OpenCoarrays-2.7.0_GNU-8.3.0/opencoarrays.mod": No such file or directory
```

**Comment by zbeekman** [comment]
I think the problem is in how I added the install logic for the sha256 install receipts. I think I have an idea of how to fix it. I'll post here when I have a patch for you guys to test. Sorry about that.

**Comment by zbeekman** [comment]
AUR bug comment: https://aur.archlinux.org/packages/opencoarrays/#comment-693351

This should be fixed in master and the 2.7.1 tag, from which I am about to publish a release.
